scss/_normalize.scss

```diff
@@ -90,41 +93,49 @@ a {
 }
 
 //
-// Improve readability of focused elements when they are also in an
-// active/hover state.
+// Remove the outline on focused links when they are also active or hovered
+// in all browsers (opinionated).
 //
 
-a {
-  &:active {
-    outline: 0;
-  }
-  &:hover {
-    outline: 0;
-  }
+a:active,
+a:hover {
+  outline-width: 0;
 }
 
 // Text-level semantics
 // ==========================================================================
 
 //
-// Address styling not present in IE 8/9/10/11, Safari, and Chrome.
+// 1. Remove the bottom border in Firefox 39-.
+// 2. Add the correct text decoration in Chrome, Edge, IE, Opera, and Safari.
 //
 
 abbr[title] {
-  border-bottom: 1px dotted;
+  border-bottom: none; // 1
+  text-decoration: underline; // 2
+  text-decoration: underline dotted; // 2
```

Comment:
This is what the original normalize looks like as well.

https://github.com/necolas/normalize.css/blob/master/normalize.css#L113